pyodide-build/pyodide_build/common.py

```diff
@@ -12,13 +12,46 @@ def _parse_package_subset(query: Optional[str]) -> Optional[Set[str]]:
     Also add the list of mandatory packages: ["pyparsing", "packaging",
     "micropip"]
 
+    Supports folowing meta-packages,
+     - 'core': corresponds to packages needed to run the core test suite
+       {"micropip", "pyparsing", "pytz", "packaging", "Jinja2"}. This is the default option
+       if query is None.
+     - 'min-scipy-stack': includes the "core" meta-package as well as some of the
+       core packages from the scientific python stack and their dependencies:
+       {"numpy", "scipy", "pandas", "matplotlib", "scikit-learn", "joblib", "pytest"}.
+       This option is non exaustive and is mainly intended to make build faster
+       while testing a diverse set of scientific packages.
+     - '*': corresponds to all packages (returns None)
+
     Returns:
       a set of package names to build or None.
```

Comment:
Would be good to say separately that `None` as an argument results from `PYODIDE_PACKAGES` being unset and leads to only `core` packages being built, and that `None` as a return value means all packages.

It might be better in the * case to return the set of all packages, which would eliminate this special case.

Reply:
> It might be better in the * case to return the set of all packages, which would eliminate this special case.

This function is only responsible for parsing CLI string input, not  loading the list of available packages from disk. So we don't have that list at this point. In the case of  all packages (`*`) we need some special marker. 

FWIW before this PR the output of None also meant to build all packages, but I agree that the fact that,
 - before: "None" input -> "None": output
 - now: "None: input -> 'core' meta package, "*" input -> "None" output

might be a bit confusing. I'll improve the docs.